include/irr/core/math/aabb2dSIMD.h

```diff
@@ -8,90 +8,92 @@
 #include <iostream>
 
 #define SIGN_FLIP_MASK_XY	core::vector3du32_SIMD(0x80000000, 0x80000000, 0x00000000, 0x00000000)
-#define SIGN_FLIP_MASK_XYZW core::vector3du32_SIMD(0x80000000, 0x80000000, 0x80000000, 0x80000000)
-
 
 namespace irr { namespace core {
 
+
 class aabb2dSIMDf
 {
 public:
 	inline aabb2dSIMDf()
-		: box( FLT_MAX, FLT_MAX, FLT_MAX, FLT_MAX ) {};
+		: internalBoxRepresentation(-FLT_MAX, -FLT_MAX, -FLT_MAX, -FLT_MAX) {};
 
-	inline aabb2dSIMDf(const core::vectorSIMDf& _box)
-		: box(_box ^ SIGN_FLIP_MASK_XY) {};
+	inline explicit aabb2dSIMDf(const core::vectorSIMDf& _box)
+		: internalBoxRepresentation(_box^ SIGN_FLIP_MASK_XY) {};
 
-	inline void addBox(const aabb2dSIMDf& _box)		
-	{ 
-		box = core::max_(box, _box.getBounds()); 
+	inline void addBox(const aabb2dSIMDf& _box)
+	{
+		internalBoxRepresentation = core::max_(internalBoxRepresentation, _box.getBounds());
 	}
 
-	inline void addPoint(const core::vectorSIMDf& point)	
+	inline void addPoint(const core::vectorSIMDf& point)
 	{
-		addBox(point); 
+		addBox(aabb2dSIMDf(point));
 	}
 
-	inline core::vectorSIMDf getBounds() const
+	inline core::vectorSIMDf getMinEdge() const
 	{
-		return box ^ SIGN_FLIP_MASK_XY;
+		core::vectorSIMDf minEdge = getBounds();
```

Comment:
be consistent and don't use `getBounds()` internally, just use `internalBoxRepresentation`